hibernate-core/src/main/java/org/hibernate/bytecode/package.html

```diff
@@ -28,9 +28,9 @@
 			</ol>
 		</p>
 		<p>
-			Note that for field-level interception, simply plugging in a new {@link BytecodeProvider}
+			Note that for field-level interception, simply plugging in a new {@link org.hibernate.bytecode.spi.BytecodeProvider}
 			is not enough for Hibernate to be able to recognize new providers.  You would additionally
-			need to make appropriate code changes to the {@link org.hibernate.intercept.Helper}
+			need to make appropriate code changes to the {@link org.hibernate.bytecode.spi.ClassLoadingStrategyHelper}
 			class.  This is because the detection of these enhanced classes is needed in a static
 			environment (i.e. outside the scope of any {@link org.hibernate.SessionFactory}.
 		</p>
```

Comment:
Again, difficult one for semantic understanding is required. Shed light on this, thanks.

Reply:
Seems fine